parsons/bill_com/bill_com.py

```diff
@@ -140,9 +170,12 @@ def get_customer_list(self, start_customer=1, max_customer=999, **kwargs):
            "max": max_customer,
            **kwargs
         }
-        return Table(self.get_request_response(data, "List", "Customer"))
 
-    def get_invoice_list(self, start_invoice=1, max_invoice=999, **kwargs):
+        r = self.get_request_response(data, "List", "Customer")
+
+        return self.paginate_list(r, data, "Customer")
```

Comment:
Not a huge deal, but curious why you aren't calling this within the `get_request_response` call instead of each individual method?